PSReadLine/Cmdlets.cs

```diff
@@ -63,6 +63,17 @@ public enum AddToHistoryOption
         MemoryAndFile
     }
 
+    public enum PredictionSource
+    {
+        None,
+        History,
+    }
+
+    public enum PredictionViewStyle
```

Comment:
I don't think this enum should be added until there is a need - right now there is no option as there is only 1 alternative.

Reply:
Fair. I will remove this enum and the corresponding option.